File: Class/1.rb
class Cal
    def initialize(v1, v2)
        @val1, @val2 = v1, v2
    end

    def add
        return @val1 + @val2
    end

    def subtract
        return @val1 - @val2
    end
end

cal1 = Cal.new(10, 10)
p cal1.add
p cal1.subtract

cal2 = Cal.new(10, 20)
p cal2.add
p cal2.subtract

File: Class/1.py
class Cal(object):
    def __init__(self, v1, v2):
        self.v1 = v1
        self.v2 = v2
    
    def add(self):
        return self.v1 + self.v2

    def subtract(self):
        return self.v1 - self.v2

cal1 = Cal(10, 10)
print(cal1.add())
print(cal1.subtract())

cal2 = Cal(10, 20)
print(cal2.add())
print(cal2.subtract())